### DISCUSSION TITLE
Change behavior of station types for rides

### DISCUSSION THREAD

**Discussion by 733737** [OpenRCT2/OpenRCT2#16511] [Feature requests]
When I was creating a park recently, I noticed that when I used object selection, I think I accidentally removed "no entrance, no station" and had to add it back, when I did this for some reason all of the stations in my park changed to canvas tent. I am not sure how this happened, I believe the game automatically selects the last station type used. I would prefer it to either select plain station or "no entrance" I also believe this is a side effect of how "no entrance, no station" is listed in object selection. The main reason I am discussing this is because although I am not a programmer, I don't believe that it would be hard to implement some kind of check before automatically changing over a dozen ride stations to canvas tent... I didn't submit this as a bug because I don't know if this is intended by the developers or not.

**Comment by duncanspumpkin** [comment]
This would happen if the order of objects changed. That should not happen. Can you reliably reproduce this?

**Reply by 733737** [reply]
i was wondering myself. I have had similar station changing automatically issues at least 3 times since the no platform option was added but never on purpose. So it isn't "rare"

**Reply by 733737** [reply]
The order of objects, could I change that accidentally if I opened the object selection menu?

**Reply by duncanspumpkin** [reply]
It should not change when accessing that menu. If it does then that is a bug.

**Reply by 733737** [reply]
How do i know when the order of objects changes? are there screen glitches or scenery menu changes?

**Reply by duncanspumpkin** [reply]
Anything that was expecting to be in a certain slot would be in a different slot so when it comes to drawing it would draw the wrong thing.

**Reply by 733737** [reply]
I use void theming a lot and I constantly run into this issue with land types for sure
